inst/htmljs/ldavis.js

```diff
@@ -584,14 +584,15 @@ LDAvis = function(to_select, json_file) {
             // create container div for topic and lambda input:
 	    var inputDiv = document.createElement("div");
 	    inputDiv.setAttribute("id", "top");
+	    inputDiv.setAttribute("style", "height: 40px; top: 10px; display: inline-block");
 
-            // insert the input container just before the vis:
-            var visDiv = document.getElementById(visID);
-            document.body.insertBefore(inputDiv, visDiv);
```

Comment:
@kshirley This, combined with `position: absolute`, causes the inputs to *always* appear at the top of the page